tests/purs/docs/src/DocCommentsMerge.purs

```diff
@@ -0,0 +1,65 @@
+module DocCommentsMerge where
```

Comment:
I guess not a good reason. I feel like the `KindSignatureDocs` properly handles the case that kind signatures are properly accounted for in all of their ways. So, it seemed like it would be better to deduplicate things, especially with #4121 coming into the picture soon.

Reply:
The DocCommentsMerge tests don’t check that comments attached to the kind signature of commentless declarations are attached to their declaration nor that comments of declarations with commentless kind signatures are preserved, like the KindSignatureDocs tests used to. I’m not sure those test are redundants, and we should perhaps even add similar ones for declarations with roles and declaration with both.

Reply:
This has been addressed in latest commits